lrauv_ignition_plugins/CMakeLists.txt

```
@@ -38,8 +38,8 @@ set(IGN_MSGS_VER ${ignition-msgs9_VERSION_MAJOR})
 find_package(ignition-plugin1 REQUIRED COMPONENTS register)
 set(IGN_PLUGIN_VER ${ignition-plugin1_VERSION_MAJOR})
```

@arjo129 I believe it should be:
```suggestion
find_package(ignition-plugin2 REQUIRED COMPONENTS register)
set(IGN_PLUGIN_VER ${ignition-plugin2_VERSION_MAJOR})
```
instead. Pushed the same fix in https://github.com/osrf/lrauv/pull/194.